src/modules/45-dashboards/pages/home/HomePage.tsx

```diff
@@ -563,311 +574,270 @@ const HomePage: React.FC = () => {
           >
-            <Layout.Horizontal
-              spacing="medium"
-              style={{ justifyContent: 'space-between', alignItems: 'center', width: '100%' }}
-            >
-              <section style={{ display: 'flex' }}>
-                <RbacButton
-                  intent="primary"
-                  text={getString('dashboardLabel')}
-                  onClick={() => showModal()}
-                  icon="plus"
-                  className={css.createButton}
-                  permission={permissionObj}
-                />
+            <section style={{ display: 'flex' }}>
+              <RbacButton
+                intent="primary"
+                text={getString('dashboardLabel')}
+                onClick={() => showModal()}
+                icon="plus"
+                className={css.createButton}
+                permission={permissionObj}
+              />
 
-                <Layout.Horizontal className={css.predefinedTags + ' ' + css.mainNavTag}>
-                  <ModuleTagsFilter selectedFilter={selectedFilter} setPredefinedFilter={setPredefinedFilter} />
-                </Layout.Horizontal>
-              </section>
+              <Layout.Horizontal className={css.predefinedTags + ' ' + css.mainNavTag}>
+                <ModuleTagsFilter selectedFilter={selectedFilter} setPredefinedFilter={setPredefinedFilter} />
+              </Layout.Horizontal>
+            </section>
+            <Layout.Horizontal>
+              <CustomSelect
+                items={sortingOptions}
+                filterable={false}
+                itemRenderer={(item, { handleClick }) => (
+                  <div key={item.value.toString()}>
+                    <Menu.Item text={item.label} onClick={handleClick} />
+                  </div>
+                )}
+                onItemSelect={item => {
+                  setSortingFilter(item)
+                }}
+                popoverProps={{ minimal: true, popoverClassName: '' }}
+              >
+                <Button
+                  inline
+                  round
+                  rightIcon="chevron-down"
+                  className={css.customSelect}
+                  text={
+                    <Layout.Horizontal spacing="xsmall">
+                      <Text color={Color.BLACK}>{'Sort By'}</Text>
+                      <Text>{sortby?.label}</Text>
+                    </Layout.Horizontal>
+                  }
+                />
+              </CustomSelect>
               <Layout.Horizontal>
-                <CustomSelect
-                  items={sortingOptions}
-                  filterable={false}
-                  itemRenderer={(item, { handleClick }) => (
-                    <div key={item.value.toString()}>
-                      <Menu.Item text={item.label} onClick={handleClick} />
-                    </div>
-                  )}
-                  onItemSelect={item => {
-                    setSortingFilter(item)
+                <Button
```

Comment:
Since there is no `text` prop or content, add the `aria-label` prop with a description of what this button does for accessibility
```suggestion
                <Button
                  aria-label={getString('switch to grid view')}
```